ste/mgr-JobPartMgr.go

```diff
@@ -13,13 +12,16 @@ import (
 	"sync/atomic"
 	"time"
 
-	"golang.org/x/sync/semaphore"
+	// there is a bug in Goland where this dependency kept getting deleted with goimport
```

Comment:
AFAIK John's made comments in the past about referencing our tooling in comments.

Perhaps just cut the comment here.

Reply:
Sure, I just wanted to justify the change